scripts/github-changelog.cr

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env crystal
+
+# This helper queries merged pull requests for a given milestone from the GitHub API
+# and creates formatted changelog entries.
+#
+# Pull requests that are already referenced in CHANGELOG.md are omitted, which
+# makes it easy to incrementally add entries.
+#
+# Entries are grouped by topic (based on topic labels) and ordered by merge date.
+# Some annotations are automatically added based on labels.
+#
+# Usage:
+#
+#   scripts/github-changelog.cr <milestone>
+#
+# Environment variables:
+#   GITHUB_TOKEN: Access token for the GitHub API (required)
+require "http/client"
+require "json"
+
+api_token = ENV["GITHUB_TOKEN"]
+repository = "crystal-lang/crystal"
+milestone = ARGV.first
+
+query = <<-GRAPHQL
+  query($milestone: String) {
+    repository(owner: $owner, name: $repository) {
+      milestones(query: $milestone, first: 1) {
+        nodes {
+          pullRequests(first: 300) {
+            nodes {
+              number
+              title
+              mergedAt
+              permalink
+              author {
+                login
+              }
+              labels(first: 10) {
+                nodes {
+                  name
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  GRAPHQL
+
+owner, _, name = repository.partition("/")
+variables = {
+  owner:      owner,
+  repository: name,
+  milestone:  milestone,
+}
+
+response = HTTP::Client.post("https://api.github.com/graphql",
+  body: {query: query, variables: variables}.to_json,
+  headers: HTTP::Headers{
+    "Authorization" => "bearer #{api_token}",
```

Comment:
```suggestion
    "Authorization" => "Bearer #{api_token}",
```

Reply:
I don't think this matters.

Reply:
https://sgeb.io/posts/2015/05/fix-go-oauth2-case-sensitive-bearer-auth-headers/#problem-statement

Reply:
> I don't think this matters.

Then you should be OK with the standard and not the deviation from it.